lib/car.rb

```diff
@@ -0,0 +1,36 @@
+class Car
+
+  def initialize(make, monthly_payment, loan_length)
+    @make = make
```

Comment:
the `make` argument in the initialize method contains both the make and the model. In the example from the interaction pattern it was "Ford Mustang", but it could be any make/model like "Toyota Camry", "Subaru Forester", etc. So what you could do is separate out the two words in the String and save the first one to `@make` and the second one to `@model`. The String#split method is helpful for breaking down a String into different parts.